intuita-webview/src/jobDiffView/DiffViewer/Collapsable.tsx

```diff
@@ -0,0 +1,65 @@
+import { forwardRef, useImperativeHandle, useState } from 'react';
+import { ReactComponent as ArrowDownIcon } from '../../assets/arrow-down.svg';
+import './Collapsable.css';
+
+type CollapsableProps = {
```

Comment:
`Readonly<...>`

Reply:
@henoktsegayetoptal 's part
Benny: addressed it